src/backend/dx12/src/command.rs

```diff
@@ -2829,6 +2829,75 @@ impl com::CommandBuffer<Backend> for CommandBuffer {
         );
     }
 
+    unsafe fn build_acceleration_structures<'a, I>(&self, _descs: I)
```

Comment:
are you sure you don't want to just make this a default implementation in the trait, to avoid all the boilerplate in backends that don't support it?

Reply:
I've undone the backend stubs for now and will make a default "unimplemented" implementation in `gfx-hal`.